packages/gorgon/src/pure-markdown.js

```diff
@@ -0,0 +1,319 @@
+// @flow
```

Comment:
I would expect this to be a top level file somewhere, not under gorgon specifically. @kevinbarabash you mentioned something about extracting this as its own package on slack?

Reply:
I can move it to its own package.